gradle-nonnull-plugin/plugin/src/test/groovy/com/novoda/gradle/nonnull/AndroidProjectIntegrationTest.groovy

```diff
@@ -0,0 +1,62 @@
+package com.novoda.gradle.nonnull
+
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.internal.DefaultGradleRunner
+import org.junit.ClassRule
+import org.junit.Test
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+import static com.google.common.truth.Truth.assertThat
+
+public class AndroidProjectIntegrationTest {
+
+    @ClassRule
+    public static final ProjectRule PROJECT = new ProjectRule()
+
+    @Test
+    public void shouldGeneratePackageAnnotation() {
+        PROJECT.generatedSrcDirs.each {
+            def file = new File(it, 'package-info.java')
```

Comment:
In my experience sometimes `def` breaks refactoring. I mentioned this as minor personal remark, probably more matter of taste. As rule of thumb I usually defer the use of `def` to those case where the type can be ambiguous (think of variants in app vs library).
Happy to keep this as it is. We probably need to agree on a common shared style for the next future - let's catch up with @devisnik soon.